userdocs/src/usage/managing-nodegroups.md

```diff
@@ -7,9 +7,11 @@ To create an additional nodegroup, use:
 ```
 eksctl create nodegroup --cluster=<clusterName> [--name=<nodegroupName>]
 ```
-
+ 
 !!!note
-    By default, new nodegroups inherit the version from the control plane (`--version=auto`), but you can specify a different
+    `--version` flag is not supported for managed nodegroup. It always inherits the version from control plane.
```

Comment:
Minor nit:
```suggestion
    `--version` flag is not supported for managed nodegroups. It always inherits the version from control plane.
```

Reply:
@cPu1 Updated the user document. Thank You